packages/scss/src/components/checkboxField/mods.scss

```diff
@@ -5,3 +5,111 @@
 	--component-checkboxField-label-fontSize: var(--pr-t-font-body-S-fontSize);
 	--component-checkboxField-label-lineHeight: var(--pr-t-font-body-S-lineHeight);
 }
+
+@mixin checklist {
```

Comment:
Another option would be to export this mod as a separate component as it won't be used in as many apps as the basic checkbox